skydive-grafana/kube/grafana-skydive.json

```diff
@@ -0,0 +1,155 @@
+{
+   "kind":"List",
```

Comment:
Two things here.. 
1. Can we get rid of the line directory
2. Can we covert to yaml? We're trying to covert all templates over to a single format